include/mbedtls/sha256.h

```diff
@@ -101,9 +102,9 @@ int mbedtls_sha256_starts_ret( mbedtls_sha256_context *ctx, int is224 );
  * \brief          This function feeds an input buffer into an ongoing
  *                 SHA-256 checksum calculation.
  *
- * \param ctx      SHA-256 context
- * \param input    buffer holding the data
- * \param ilen     length of the input data
+ * \param ctx      The SHA-256 context to initialize.
```

Comment:
I believe that the context is not initialized here, it just carries the data necessary to conduct the checksum calculation.
Is that right, @hanno-arm?

Reply:
The function description seems to support your assumption. Removed unless Hanno advises otherwise.